src/main/java/org/jenkinsci/test/acceptance/plugins/git/GitScm.java

```diff
@@ -103,6 +111,16 @@ public GitScm calculateChangelog(String remote, String branch) {
     }
 
     /**
+     * Add behaviour "Use commit author in changelog"
```

Comment:
Do we need all these "Add behaviour" wrapper methods? To me, `addBehaviour(CommitAuthorInChangelog.class)` is readable enough to put into test case directly...

Reply:
I agree with you about `addBehaviour` method is readable enough. Since some versions ago, the UI has suffered many changes and they could still change more. If we use the `addBehaviour` method and those behaviours change, all the test involved will have to be updated. I'm trying to isolate as much as possible test from all these possible changes since class `GitScm` will have to be updated in that case. It could be helpful in the future (IMHO)